ruby_event_store/lib/ruby_event_store/mappers/default.rb

```diff
@@ -0,0 +1,35 @@
+require 'yaml'
+require 'active_support'
+
+module RubyEventStore
+  module Mappers
+    class Default
+      def initialize(serializer: YAML, events_class_remapping: {})
+        @serializer = serializer
+        @events_class_remapping = events_class_remapping
+      end
+
+      def event_to_serialized_record(domain_event)
+        SerializedRecord.new(
+          id:         domain_event.event_id,
+          metadata:   serializer.dump(domain_event.metadata),
+          data:       serializer.dump(domain_event.data),
+          event_type: domain_event.class
```

Comment:
That should be a `String`. Very important!

Reply:
Thats why I like contracts.ruby, this can be detected on first run of code (in spec)